docs/preview/features/powershell/azure-security.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Scripts related to Azure security"
+layout: default
+---
+
+# Azure Security
+
+This module provides the following capabilities:
+- [Removing resource locks on azure](#removing-resource-locks-on-azure)
```

Comment:
```suggestion
- [Removing resource locks from an Azure resource group](#removing-resource-locks-from-an-azure-resource-group)
```
So it clearly states it's about removing locks from a specific resource group, not removing locks in general.